libutempter/Makefile

```diff
@@ -21,6 +21,8 @@
 PROJECT = utempter
 VERSION = $(shell sed '/^Version: */!d;s///;q' libutempter.spec)
 MAJOR = 0
+RMFLAGS = -fr
```

Comment:
Why the recursive flag? I see no directories removed.

Reply:
In the development process, I have directories store line coverage data, but I forgot to remove -r flag. Now I remove them